code/modules/clothing/clothing.dm

```diff
@@ -198,6 +200,27 @@ BLIND     // can't see anything
 		var/mob/M = loc
 		M.update_inv_shoes()
 
+/obj/item/clothing/shoes/attackby(obj/item/I, mob/user, params)
+	if(!can_hold_knives)
+		return
+	if(held_knife)
+		user << "<span class='notice'>There's already something in [src].</span>"
+		return
+	if(istype(I, /obj/item/weapon/kitchen/knife) || istype(I, /obj/item/weapon/pen))//can hold both regular pens and energy daggers. made for your every-day tactical librarians/murderers.

```

Comment:
i considered it but i figured that would be too much. it would be like having a third pocket that can't be pickpocketed and this feature isn't supposed to be THAT powerful.